### ISSUE TITLE
Two crashes in a row in one session

### ISSUE THREAD

**Issue by shadowpanther** [Yellow-Dog-Man/Resonite-Issues#407]
### Describe the bug?

I got two similar crashes yesterday shortly after joining one particular session.

### To Reproduce

Was consistent for me on joining one certain session, not sure what object/asset in the session caused it.

### Expected behavior

Not to crash

### Resonite Version Number

2023.10.20.831

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Quest 3 (Oculus runtime)

### Log Files

[Crash_2023-10-22_174518248.zip](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13074965/Crash_2023-10-22_174518248.zip)
[VIXEN - 2023.10.20.831 - 2023-10-22 19_41_55.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13075028/VIXEN.-.2023.10.20.831.-.2023-10-22.19_41_55.log)
[VIXEN - 2023.10.20.831 - 2023-10-22 19_45_57.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13075032/VIXEN.-.2023.10.20.831.-.2023-10-22.19_45_57.log)

### Additional Context

CPU: Core i7-11700
RAM: 32Gb
GPU: Radeon RX 6900 XT

Might be related to #269 

### Reporters

Shadow Panther

**Comment by shadowpanther** [comment]
Other users were fine (there were 3 users, I think, on a headless host).
I crashed about a minute after joining both times. I was able to load all resources and start the conversation when the crash happened both times.
I didn't try this session again after that because restarting the whole client is a bit tedious.
I joined the session much later that evening with at least one of the original users gone and didn't crash. Maybe the objects present in the world were added/deleted too.

From what I know, Unity crashes are pretty much random, but the probability increases in certain scenarios (a lot of video players, RenderTransformOverrides, etc.).
If there's a methodology to somehow identify the crashing factor before crashing, I'd like to know it. Otherwise, asking people to leave the session one by one while restarting my client is not very feasible in social setting.
If there's a setting I can turn on to get more verbose logs before crash, I can use that too.
